### ISSUE TITLE
5.0.1 in Maven?

### ISSUE THREAD

**Issue by CosminCH** [qooxdoo/qooxdoo#9052]
Hello,

Do you have any plans to expose the latest version (5.0.1) as a Maven artifact?

Thank you.
Cosmin

**Comment by johnspackman** [comment]
I don't think so, but we would welcome a PR which included it.  

Are these pages any use?

http://qooxdoo.org/docs/general/maven_integration
http://qooxdoo.678.n2.nabble.com/Maven-integration-td7563674.html
https://github.com/salmon-charles/qooxdoo-maven

**Comment by CosminCH** [comment]
Not really, I do have a workaround. Still, it should be published on Maven, as all the other versions.
